k8s/upgrades/README.md

```diff
@@ -226,10 +231,14 @@ spec:
         - "--from-version=1.0.0"
 
         # --to-version is the version desired upgrade version
-        - "--to-version=1.8.0"
+        - "--to-version=1.9.0"
 
-        #VERIFY that you have provided the correct cStor PV Name
+        # VERIFY that you have provided the correct jiva PV Name
         - "--pv-name=pvc-713e3bb6-afd2-11e9-8e79-42010a800065"
+        # From 1.9.0 onwards the pv-name flag is not required and 
```

Comment:
you can mention just this one, and outside of this sample yaml, mention that '--pv-name' is also supported for backward compatibility for few more releases

Reply:
same comment applies for other places also
Also, need to mention how to read the job output to know the status of individual volume's upgrade